addons/mail/security/ir.model.access.csv

```diff
@@ -10,6 +10,7 @@ access_mail_followers_system,mail.followers.system,model_mail_followers,base.gro
 access_mail_notification_portal,mail.notification.portal,model_mail_notification,base.group_portal,1,0,0,0
 access_mail_notification_user,mail.notification.user,model_mail_notification,base.group_user,1,1,1,0
 access_mail_notification_system, mail.notification.system,model_mail_notification,base.group_system,1,1,1,1
+access_discuss_channel_category,discuss.channel.category,model_discuss_channel_category,base.group_user,1,1,1,1

```

Comment:
If you don't add access rules, this means all users can freely add/remove/edit categories.
Wouldn't we want to only allow them to admins?
Or at least, a specific discuss-manager group?

Reply:
I think it might be time to create this group for discuss?
There are several features I feel are not worth being restricted to admin only, while a specific discuss admin would work best.